WordPress/Sniffs/PHP/DiscouragedPHPFunctionsSniff.php

```diff
@@ -97,6 +97,16 @@ public function getGroups() {
 					'str_rot13',
 				),
 			),
+			'typecasting' => array(
+				'type'      => 'warning',
+				'message'   => '%s() can be replaced with type casting.',
```

Comment:
Should this message be expanded a little to explain why ?

Maybe something along the lines of _"`%s()` should only be used in callbacks. In all other cases, using type casting will be more performant."_

Reply:
Updated to this text in 2aeb355